src/Language/PureScript/Linter/Exhaustive.hs

```diff
@@ -242,8 +242,6 @@ checkExhaustive env mn cas = makeResult . first nub $ foldl' step ([initial], (p
 
   initial :: [Binder]
   initial = initialize numArgs
```

Comment:
If you can just remove this, and the function argument from `initialize`, I'll merge this. Thanks!